styles/Stars.module.css

```diff
@@ -7,9 +7,15 @@
   width: 45px;
   overflow: unset;
   margin-right: 15px;
+  color: gray;
 }
 .star:hover {
-  opacity: 0.5;
+  cursor: pointer;
+  color: #a98d00;
+}
+
+.selected {
+  color: #ddb802;
```

Comment:
This is nitpicky, but I think I would choose a slightly lighter grey for the deselected stars (one that's still AA compliant for contrast!) I think I'd also  prefer to see this darker yellow on the `selected:hover` but maybe have a darker grey on the `star:hover`? Happy to defer to the second reviewer on this!